pandas/tests/test_downstream.py

```diff
@@ -123,6 +123,7 @@ def test_geopandas():
     assert geopandas.read_file(fp) is not None
 
 
+@pytest.mark.xfail(reason="ix-only methods have been removed from _NDFrameIndexer")
```

Comment:
cc @jorisvandenbossche 

@jbrockmendel can you open an issue on geopandas tracker?

Reply:
Vague plan is to wait for Joris to get back and advise on how to proceed

Reply:
There is already an open issue for that: https://github.com/geopandas/geopandas/issues/1042

My advice is to keep compatibility with released geopandas for now (see the comment I made above on a specific method of NDFrameIndexer). I know geopandas shouldn't be using this (I opened a PR to clean it up: https://github.com/geopandas/geopandas/pull/1072), but it is, and given we know that, it would be nice to not break directly.